app/models/megaphone/episode.rb

```diff
@@ -163,11 +187,8 @@ def check_audio!
       # Re-request the megaphone api
       find_by_megaphone_id
 
-      # get the audio attributes from feeder
-      set_audio_attributes
```

Comment:
Don't set these, it overrides the values from megaphone with the values in feeder, and we want to check what's in megaphone!